### PR TITLE
small documentation fixes

### PR DESCRIPTION
**Pull request by hohwille**
doc changes that should have been done on master but accidentally happened on develop.

### CODE REVIEW

**Comment by hohwille** [comment]
Seems as if we have to merge this manually. Does not work via PR and includes unwanted changes.

File: documentation/guide-accessibility.asciidoc [hunk 1]
```diff
@@ -3,6 +3,13 @@ toc::[]
...
 = Accessibility
 
+ATTENTION: You are on the `develop` branch.
+This has been renamed to `master`.
+The `develop` branch will not be maintained anymore.
+It is only left here to avoid broken links to existing content.
+Please update links to point to the `master` branch.
+For details look at issue https://github.com/devonfw/devon4j/issues/320[#320].
+
 TODO
```
Comment (maybeec) [inline documentation/guide-accessibility.asciidoc:13]: What about this?

File: README.adoc [hunk 2]
```diff
@@ -1,5 +1,12 @@
 = devon4j
 
+ATTENTION: You are on the `develop` branch. 
+This has been renamed to `master`. 
+The `develop` branch will not be maintained anymore.
+It is only left here to avoid broken links to existing content.
+Please update links to point to the `master` branch.
+For details look at issue https://github.com/devonfw/devon4j/issues/320[#320].
+
```
Comment (maybeec) [inline README.adoc:9]: To whom do you provide this information?
What is the intention why a devonfw customer has to know such internal stuff?

**Review by maybeec — CHANGES_REQUESTED** [review]

File: README.adoc (hunk 2 above)
Comment (hohwille) [reply README.adoc:9]: This is from isse #320.
It should have prevented this entire PR but still I sometimes had been mislead by old browser history and URL auto-completion leading me to the wrong branch...

**Comment by hohwille** [comment]
I have created a branch protection rule for `develop` that prevents such changes and that this may ever happen again in the future.

**Comment by hohwille** [comment]
merged all relevant changes manually and prevented that such changes can happen on the wrong branch again.
Hence closing.

**Pull request closed without merging**